packages/files/file.pony

```diff
@@ -412,81 +533,93 @@ class File
     let pos = position()
     _seek(0, 2)
     let len = position()
-    _seek(pos.isize(), 0)
+    _seek(pos.i64(), 0)
     len
 
-  fun ref seek_start(offset: USize) =>
+  fun ref seek_start(offset: USize): File =>
```

Comment:
Hmmmm... This is an oversight on my part. The changes were made in our fork prior to the chaining operator and I didn't notice that when I copied them over. I'll fix it.